docs/casa/plugins/cert-authn-tutorial.md

```
@@ -0,0 +1,97 @@
+---
+tags:
+- Casa
+- 2FA
+- Certificate
+- Smartcards
+- Yubikeys
+---
+
+# Tutorial: smart card authentication in Janssen
+
+With Casa's [certificate authentication plugin](./cert-authn.md) administrators can configure client certificate authentication so users can present digital certificates as a form of second-factor authentication. Some organizations opt to deploy certificate authentication through the use of smart cards - physical cards with embedded integrated circuits that act as security tokens. This practice is considered to offer a high level of security compared to other forms of multi-factor authentication. A well known example of smart card usage is the the U.S. Department of Defense (DoD). 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)